### PR TITLE
Added configuration example for feature toggle.

### PR DESCRIPTION
**Pull request by ivanmrsulja**
[Vitro PR](https://github.com/vivo-project/Vitro/pull/421)
# What does this pull request do?
Added configuration example of forgotPassword functionality feature toggle.

# What's new?
Added example of configuration in example.runtime.properties file.

# How should this be tested?
this should be tested together with the corresponding [Vitro PR](https://github.com/vivo-project/Vitro/pull/421).

# Interested parties
@chenejac @brianjlowe @litvinovg

### CODE REVIEW

**Review by chenejac — APPROVED** [review]
@ivanmrsulja thanks

File: home/src/main/resources/config/example.runtime.properties [hunk 1]
```diff
@@ -455,3 +455,6 @@ createAndLink.providers = doi, pmid
 #fileUpload.maxFileSize = 10485760
 #comma separated list of mime types allowed for upload
 #fileUpload.allowedMIMETypes = image/png, application/pdf
+
+# Feature toggle for forgot password functionality
+authentication.forgotPassword = enabled
```
Comment (chenejac) [inline home/src/main/resources/config/example.runtime.properties:460]:
```suggestion
authentication.forgotPassword = enabled
authentication.forgotPassword.notify-admin = true
```

**Review by chenejac — CHANGES_REQUESTED** [review]

**Comment by litvinovg** [comment]
@ivanmrsulja please resolve merge conflict

File: home/src/main/resources/config/example.runtime.properties [hunk 2]
```diff
@@ -456,6 +456,10 @@ createAndLink.providers = doi, pmid
 #comma separated list of mime types allowed for upload
 #fileUpload.allowedMIMETypes = image/png, application/pdf
 
+# Feature toggle for forgot password functionality
+authentication.forgotPassword = false
```
Comment (chenejac) [inline home/src/main/resources/config/example.runtime.properties:460]:
```suggestion
authentication.forgotPassword = disabled
```

**Review by chenejac — CHANGES_REQUESTED** [review]

File: home/src/main/resources/config/example.runtime.properties [hunk 3]
```diff
@@ -456,6 +456,10 @@ createAndLink.providers = doi, pmid
 #comma separated list of mime types allowed for upload
 #fileUpload.allowedMIMETypes = image/png, application/pdf
 
+# Feature toggle for forgot password functionality
```
Comment (wwelling) [inline home/src/main/resources/config/example.runtime.properties:459]: We probably want to provide the available options here. I assume `[enabled, disabled]`.

**Review by chenejac — APPROVED** [review]

**Review by wwelling — APPROVED** [review]

**Review by litvinovg — APPROVED** [review]

**Pull request merged**